bindings/trycp_runner/src/common.rs

```diff
@@ -47,7 +47,7 @@ pub fn connect_trycp_client<SV: UserValuesConstraint>(
 
     let client = ctx.runner_context().executor().execute_in_place({
         let signer = signer.clone();
-        async move { Ok(TryCPClient::connect(target, signer.clone(), reporter).await?) }
+        async move { TryCPClient::connect(target, signer.clone(), reporter).await.with_context(|| format!("Could not connect TryCP client, is there a server running and reachable at {}?", target)) }
```

Comment:
"Evidently not" 😉

Reply:
Haha, just trying to be helpful. Connection refused isn't very meaningful. 

I do this regularly, so I have to be nice to myself in the messages ;)